android/src/com/mapswithme/maps/editor/LanguagesFragment.java

```diff
@@ -17,7 +17,8 @@
 
 public class LanguagesFragment extends BaseMwmRecyclerFragment
 {
-  final static String EXISTING_LOCALIZED_NAMES = "ExistingLocalizedNames";
+  public static final String EXISTING_LOCALIZED_NAMES = "ExistingLocalizedNames";
+  private static final String DEFAULT_LANG_CODE = "default";
```

Comment:
Please add a comment that "default" value mimics the one from StringUtf8Multilang.